### PR TITLE
[14.0][MIG] account_invoice_line_sequence

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Provides a new sequence field on invoice lines which helps to manage the order of the invoice lines.

The sequence number always starts at 1 and rises incrementally: 1, 2, 3, ... To keep it that way, all sequence numbers are recalculated when the order of invoice lines is changed. The line number is printed on the invoice report.

### CODE REVIEW

**Review by andreampiovesana — APPROVED** [review]
go

**Review by matteoopenf — APPROVED** [review]

**Review by andreampiovesana — APPROVED** [review]
merge?

**Comment by matteoopenf** [comment]
@OCA/accounting-maintainers

File: account_invoice_line_sequence/models/account_move.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    sequence = fields.Integer(
+    sequence_show = fields.Integer(
```
Comment (MiquelRForgeFlow) [inline account_invoice_line_sequence/models/account_move.py]: this is wrong, don't rename it. it's intended to rewrite the original sequence field.

File: account_invoice_line_sequence/models/account_move.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    sequence_show = fields.Integer(
+        help="Shows the sequence of this line in the " " invoice.",
```
Comment (MiquelRForgeFlow) [inline account_invoice_line_sequence/models/account_move.py]:
```suggestion
        help="Shows the sequence of this line in the invoice.",
```
please, do this in all the cases where it happens

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: account_invoice_line_sequence/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Damien Crier <damien.crier@camptocamp.com>
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
```
Comment (MiquelRForgeFlow) [inline account_invoice_line_sequence/readme/CONTRIBUTORS.rst]: 😅

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review, it looks good, including the printout

**Pull request merged**